makelib/kubebuilder.mk

```diff
@@ -25,6 +26,38 @@ TEST_ASSET_ETCD := $(KUBEBUILDER)/etcd
 TEST_ASSET_KUBECTL := $(KUBEBUILDER)/kubectl
 export TEST_ASSET_KUBE_APISERVER TEST_ASSET_ETCD TEST_ASSET_KUBECTL
 
+# ====================================================================================
+# Setup go environment
+
+-include golang.mk
+
+ifeq ($(CRD_DIR),)
+$(error please set CRD_DIR prior to including kubebuilder.mk)
+endif
+
+ifeq ($(CONTROLLERGEN_PATHS),)
+$(error please set CONTROLLERGEN_PATHS prior to including kubebuilder.mk)
+endif
+
+
+CONTROLLERGEN := $(TOOLS_HOST_DIR)/controller-gen
+
+# ====================================================================================
+# Kubebuilder Targets
+
+# Generate manifests e.g. CRD, RBAC etc.
+kubebuilder.manifests: $(CONTROLLERGEN)
+	@$(INFO) Generating CRD manifests
+	@$(CONTROLLERGEN) crd:trivialVersions=true paths=$(CONTROLLERGEN_PATHS) output:dir=$(CRD_DIR)
```

Comment:
Could you add a `rm -rf $(CRD_DIR)` before generation call? Because in its current state, generation doesn't delete the CRDs that do not exist anymore but only change the existing ones and add the new ones.

Reply:
added the line